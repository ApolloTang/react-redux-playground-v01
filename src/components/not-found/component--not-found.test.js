import React from 'react';
import NotFound from './index';
import testRenderer from 'react-test-renderer';

describe(`
  *********************************************
  **             Component Test              **
  **                not-found                **
  *********************************************
  `, () => {
    test(`:::: snapshot test `, () =>{
      const component = testRenderer.create(<NotFound />);
      const componentJson = component.toJSON();
      expect(componentJson).toMatchSnapshot();
    });
});




