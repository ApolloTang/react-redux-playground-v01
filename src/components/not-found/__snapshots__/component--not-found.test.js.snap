// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`
  *********************************************
  **             Component Test              **
  **                not-found                **
  *********************************************
   :::: snapshot test  1`] = `
<div
  className="langindPage module-style"
>
  Not found
</div>
`;
